### PR TITLE
Implement missing packet and define some unk for Trophy System

### PR DESCRIPTION
**Pull request by Mesielecat**

### CODE REVIEW

File: WowPacketParser/WowPacketParser.csproj [hunk 1]
```diff
@@ -385,6 +385,8 @@
     <Compile Include="Enums\TrainerSpellState.cs" />
     <Compile Include="Enums\TrainerType.cs" />
     <Compile Include="Enums\TransferAbortReason.cs" />
+	 <Compile Include="Enums\TrophyLockCode.cs" />
```
Comment (mdX7) [inline WowPacketParser/WowPacketParser.csproj]: tab

File: WowPacketParser/Enums/TrophyLockCode.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace WowPacketParser.Enums
+{
+    public enum TrophyLockCode
+    {
+        CodeWrongAchievement = 34,
```
Comment (mdX7) [inline WowPacketParser/Enums/TrophyLockCode.cs]: remove "Code" term in CodeWrongAchievement and CodeSuccess. This is not C++, enums here kinda work the same way as `enum class` in C++, that means the enum name itself is required - e.g. TrophyLockCode.WrongAchievement.

File: WowPacketParser/Enums/TrophyType.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿namespace WowPacketParser.Enums
+{
+    public enum TrophyType
+    {
+        TrophyHorde    = 3,
```
Comment (mdX7) [inline WowPacketParser/Enums/TrophyType.cs]: remove "Trophy" term

**Comment by Mesielecat** [comment]
kk , gonna fix that soon

**Pull request merged**